components/cam/bld/configure

```diff
@@ -144,7 +143,7 @@ OPTIONS
                         se grids.
      -max_n_rad_cnst <n> Maximum number of constituents that are either radiatively
                         active, or in any single diagnostic list for the radiation.    
-     -microphys <name>  Specify the microphysics option [mg1 | mg1.5 | mg2 | rk | p3].
```

Comment:
We will probably need this change to stay in SCREAM for @bogensch work on the SCREAM Physics Compset

Reply:
we need to retain this line